.editorconfig

```diff
@@ -0,0 +1,77 @@
+root = true
+
+[*]
+indent_style = tab:suggestion
+insert_final_newline = true:suggestion
+trim_trailing_whitespace = true:suggestion
+
+[*.cs]
+csharp_prefer_braces = true:suggestion
+csharp_new_line_before_open_brace = none:suggestion
+csharp_new_line_before_else = false:suggestion
+csharp_new_line_before_catch = false:suggestion
+csharp_new_line_before_finally = false:suggestion
+csharp_new_line_before_members_in_object_initializers = false:suggestion
+csharp_new_line_before_members_in_anonymous_types = false:suggestion
+csharp_new_line_between_query_expression_clauses = false:suggestion
+
+csharp_style_var_for_built_in_types = false:suggestion
+csharp_style_var_when_type_is_apparent = false:suggestion
+csharp_style_var_elsewhere = false:suggestion
+
+csharp_style_expression_bodied_methods = when_on_single_line:suggestion
+csharp_style_expression_bodied_constructors = when_on_single_line:suggestion
+csharp_style_expression_bodied_operators = when_on_single_line:suggestion
+csharp_style_expression_bodied_properties = when_on_single_line:suggestion
+csharp_style_expression_bodied_indexers = when_on_single_line:suggestion
+csharp_style_expression_bodied_accessors = when_on_single_line:suggestion
+
+csharp_style_pattern_matching_over_as_with_null_check = false:suggestion
+
+csharp_preferred_modifier_order = public,private,protected,internal,new,abstract,virtual,sealed,override,static,readonly,extern,unsafe,volatile,async:suggestion
```

Comment:
Use official `public,private,protected,internal,static,extern,new,virtual,abstract,sealed,override,readonly,unsafe,volatile,async` instead, like I pointed out above, keeping suggestion severity.

Reply:
Actually I wonder if this is even right, I should look it up in my R# manually, this doesn't seem fine.

Reply:
![image](https://user-images.githubusercontent.com/6608231/42510564-97d8c368-844f-11e8-83f4-0e9373cf8854.png)

Reply:
![image](https://user-images.githubusercontent.com/1069029/42510652-e6ed7084-844f-11e8-8027-90bfbd9f13a2.png)

Ah right, it's per-line and not per-file, then it's right, we want defaults like I said above, R# defaults are very close to them since the groups match.

Reply:
yes. sadly we can't specify an order per file.

Reply:
Yes, but we should still use C# defaults for that and not R# defaults - https://github.com/JustArchi/ArchiSteamFarm/commit/2a1d5d1e60a7e72198fda4aafe01d5eb56dbd21c

Reply:
then we'll do that.